lasio/writer.py

```diff
@@ -110,7 +110,12 @@ def write(
     if STEP is None:
         if STOP != STRT:
             # prevents an error being thrown in the case of only a single sample being written
-            STEP = las.index[1] - las.index[0]  # Faster than np.gradient
+            # Example: 0.51 -> "51" -> len("51") -> 2
```

Comment:
Another option is to use the writer.write() parameter: `fmt="%.5f",` instead of calculating the length.

Reply:
I've make a 2nd pull request #432 that standardizes output on `fmt="%.5f"` for comparison.   It currently fails on some older tests because it changes the fractional precision of the STEP value.

Reply:
Tests in #432 have been updated to pass.